mava/systems/ff_ippo.py

```diff
@@ -510,7 +510,7 @@ def run_experiment(_config: DictConfig) -> float:
         )
 
     # Run experiment for a total number of evaluations.
-    max_episode_return = jnp.float32(0.0)
+    max_episode_return = jnp.float32(-jnp.inf)
```

Comment:
We don't actually need the `jnp.float32` part :smile: 

```suggestion
    max_episode_return = -jnp.inf
```